camelot/admin/action/list_action.py

```diff
@@ -144,10 +144,23 @@ class OpenFormView(Action):
     verbose_name = None
     name = 'open_form_view'
 
+    def get_object(self, model_context, mode):
+        if mode is None:
```

Comment:
ik zou forceren dat het dan de mode gebruikt ? meerdere manieren om hetzelfde te doen is altijd vodden